src/cockpit/beiboot.py

```diff
@@ -175,10 +175,16 @@ async def do_askpass(self, messages: str, prompt: str, hint: str) -> Optional[st
         host_match = re.search(r"authenticity of host '([^ ]+) ", prompt)
         args = {}
         if fp_match and host_match:
+            hostname = host_match.group(1)
+            # common case: don't ask for localhost's host key
+            if hostname == '127.0.0.1':
```

Comment:
What happens if we don't accept this silently? Right now I think we shouldn't.

Reply:
The user would get a host key prompt for each initial login with the cockpit/ws container. We could work around that in the tests of course, but it's an awkward user experience. We've had that behaviour for many years, and I didn't want to change it now -- that'd be pretty dramatic now. Just like the beiboot disabling, I don't want to change observable behaviour with this PR, it's intrusive enough already.

Reply:
That is correct. Of course the user *could* type an explicit "Connect to: 127.0.0.1", and with both c-ssh and c.beiboot it still wouldn't ask for a host key confirmation.

Note that there actually *is* a behaviour change here: If cockpit.conf sets a non-default `Host=` value in [Ssh-Login], then cockpit-ssh would *not* check the host key. I find that dangerous and also an abuse of the original intent of that option (it's originally read by cockpit-ws), I couldn't get myself to reimplementing that. I tried to describe it in the commit message.